lib/formtools/overview.js

```diff
@@ -0,0 +1,183 @@
+
+var async = require('async'),
+    cellRenderers = require('./renderers-cell');
+
+/**
+ * Set unique tabId to each tab in overview.body
+ * @param {Array} sections overview body array
+ */
+function setTabId (sections) {
+
+    sections.forEach(function (section, sectionIndex) {
+
+        if (!Array.isArray(section)) {
+            return;
+        }
+
+        section.forEach(function (tab, tabIndex) {
+
+            // Replace white space from tab.lable with "-", make it lowercase and add the index number at the end.
+            // sectionIndex and tabIndex number are added at the end to make the tabID unique in case when more then one tab have the same label (title)
+            tab.tabId = (tab.label).replace(/\s+/g, '-').toLowerCase() + '-' + sectionIndex + '-' + tabIndex;
+
+        });
+
+    });
+
+}
+
+/*
+ * Update section.fields to an array of objects with label and value keys. Label should contain the field label and value should contain the field value.
```

Comment:
Can you please update this sentence to the below for more clarity:
`Update section.fields to an array of objects with 'label' and 'value' keys. 'label' key should contain the field label and 'value' key should contain the field value.`